integration/screen_widgets_test.go

```diff
@@ -28,9 +29,7 @@ func TestWidgetAlertValue(t *testing.T) {
 		AddTimeframe: datadog.Bool(false),
 	}
 
-	w := datadog.Widget{AlertValueWidget: expected}
-
-	board.Widgets = append(board.Widgets, w)
+	board.Widgets = append(board.Widgets, expected)
```

Comment:
`board.Widgets` is not the same type, so this cause tests to fail. My first thought on fixing it is doing something like:

```
var widgetInterface interface{} = expected
widget, ok := widgetInterface.(datadog.Widget)
if !ok {
	t.Fatalf("TODO: some actually useful message here...")
}

board.Widgets = append(board.Widgets, widget)
```

I haven't had time to fix this everywhere to see if there are other issues.